.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
`https://travis-ci.org/xm-online/xm-ms-config/builds/628660155?utm_source=github_status&utm_medium=notification`

can't see information about real test execution. Please check this.
Expceted something like this
`com.icthh.xm.ms.configuration.service.processors.TenantConfigExternalizationUnitTes > overrideDoubleNumbers PASSED`

Reply:
Done